documentation/streaming-search.html

```diff
@@ -82,29 +46,112 @@
+sample apps</a>:
+<ol>
+  <li>
+    Set indexing mode to <a href="reference/services-content.html#document">streaming</a>:
 <pre>
 &lt;content id="mycluster" version="1.0"&gt;
   &lt;documents&gt;
-    &lt;document type="mytype" <span style="background-color: yellow;">mode="streaming"</span> /&gt;
+    &lt;document type="music" <span style="background-color: yellow;">mode="streaming"</span> /&gt;
 </pre>
-To run a streaming search query: <!-- ToDo: rewrite - use group or userid, not both -->
-<ul>
-  <li>specify the subset group membership of every document in the document id,
-    see groupname / user id in <a href="documents.html">document IDs</a></li>
-  <li>specify the subset to search in every query using the query request attribute
+  </li><li>
+    <a href="documents.html">Document IDs</a> must have a numeric id or string for the set of documents to search -
+    numeric example:
+<pre>
+$ curl -H "Content-Type:application/json" \
+  --data-binary @feedfile.json \
+  http://localhost:8080/document/v1/mynamespace/music/<span style="background-color: yellow">docid</span>/1
+
+$ curl -H "Content-Type:application/json" \
+  --data-binary @feedfile.json \
+  http://localhost:8080/document/v1/mynamespace/music/<span style="background-color: yellow">number/1234</span>/1
+</pre>
+  </li><li>
+    Specify the subset to search using the query request attribute
     <a href="reference/query-api-reference.html#streaming.groupname">streaming.groupname</a> or
-    <a href="reference/query-api-reference.html#streaming.userid">streaming.userid</a></li>
-</ul>
-Example: <em>http://hostname.com:8080/search?query=test&amp;hits=20&amp;streaming.userid=12345678&amp;timeout=10</em>
+    <a href="reference/query-api-reference.html#streaming.userid">streaming.userid</a>.
+    Example (URL decoded for readability):
+<pre>
+http://localhost:8080/search?yql=select * from sources * where artist contains "coldplay";&amp;<span style="background-color: yellow">streaming.userid=1234</span>
+</pre>
+  </li>
+</ol>
+Streaming mode is search in document store data.
+Changing from index to streaming mode hence requires refeeding all documents.
+Use <a href="reference/vespa-cmdline-tools.html#vespa-remove-index">vespa-remove-index</a>
+to drop documents on nodes before changing mode.
+</p>
+
+
+<h3 id="match-mode">Match mode</h3>
+<p>
+Next step is setting correct <a href="reference/schema-reference.html#match">match mode</a> for fields -
+example using default string tokenized matching:
+<pre>
+field artist type string {
+    indexing: summary | index
+}
+
+# Use <a href="reference/vespa-cmdline-tools.html#vespa-configproxy-cmd">vespa-configproxy-cmd</a> to find the value for the -i argument below
+
+$ <a href="reference/vespa-cmdline-tools.html#vespa-get-config">vespa-get-config</a> -n vespa.config.search.vsm.vsmfields -i music/search/cluster.music.music | egrep 'name|arg1'
```

Comment:
I am not sure we want to put this information here. This is for debugging and might confuse the reader. What is relevant is what we put in "match:" in the field definition. The same applies for the examples further down. Perhaps add a debug section at the bottom with one example using vespa-get-config.

Reply:
good point! that makes it more readable. I included it here to make it easier for people to see actual settings. will change in next PR